AutomatedTesting/Gem/PythonTests/scripting/ScriptCanvasComponent_OnEntityActivatedDeactivated_PrintMessage.py

```diff
@@ -5,47 +5,58 @@
 SPDX-License-Identifier: Apache-2.0 OR MIT
 """
 
-import os
-import editor_python_test_tools.pyside_utils as pyside_utils
-import editor_python_test_tools.hydra_editor_utils as hydra
-from editor_python_test_tools.utils import Report, Tracer
-from editor_python_test_tools.utils import TestHelper as helper
-import scripting_utils.scripting_tools as scripting_tools
-import azlmbr.legacy.general as general
-import azlmbr.paths as paths
-from scripting_utils.scripting_constants import (WAIT_TIME_3, BASE_LEVEL_NAME)
-
-EXPECTED_LINES = ["Activator Script: Activated", "Deactivator Script: Deactivated"]
-controller_dict = {
-    "name": "Controller",
-    "status": "active",
-    "path": os.path.join(paths.projectroot, "ScriptCanvas", "OnEntityActivatedScripts", "controller.scriptcanvas"),
-}
-activated_dict = {
-    "name": "ActivationTest",
-    "status": "inactive",
-    "path": os.path.join(paths.projectroot, "ScriptCanvas", "OnEntityActivatedScripts", "activator.scriptcanvas"),
-}
-deactivated_dict = {
-    "name": "DeactivationTest",
-    "status": "active",
-    "path": os.path.join(paths.projectroot, "ScriptCanvas", "OnEntityActivatedScripts", "deactivator.scriptcanvas"),
-}
-ENTITY_TO_ACTIVATE_PATH = "Configuration|Properties|Variables|EntityToActivate|Datum|Datum|value|EntityToActivate"
-ENTITY_TO_DEACTIVATE_PATH = "Configuration|Properties|Variables|EntityToDeactivate|Datum|Datum|value|EntityToDeactivate"
-WAIT_ONE_SECOND = 1.0
+
 # fmt: off
 class Tests():
-    controller_exists    = ("Successfully found controller entity",  "Failed to find controller entity")
-    activated_exists     = ("Successfully found activated entity",   "Failed to find activated entity")
-    deactivated_exists   = ("Successfully found deactivated entity", "Failed to find deactivated entity")
-    start_states_correct = ("Start states set up successfully",      "Start states set up incorrectly")
     game_mode_entered    = ("Successfully entered game mode",         "Failed to enter game mode")
     lines_found          = ("Successfully found expected prints",    "Failed to find expected prints")
     game_mode_exited     = ("Successfully exited game mode", "Failed to exit game mode")
 # fmt: on
 
-class ScriptCanvasComponent_OnEntityActivatedDeactivated_PrintMessage():
+
+def setup_level_entities(dictionaries):
```

Comment:
rename dictionaries to entities

Reply:
renamed the container as it now contains the test entities objects